dist/windows/installer-translations/czech.nsi

```diff
@@ -1,4 +1,4 @@
-﻿;Installer strings
+;Installer strings
```

Comment:
What is actually changed in this line? It looks like it might be something undesirable...

Reply:
UTF-8 BOM is removed.